modules/apps/callbacks/testing/simapp/app.go

```diff
@@ -522,7 +522,7 @@ func NewSimApp(
 
 	// Create Transfer Stack
 	// SendPacket, since it is originating from the application to core IBC:
-	// transferKeeper.SendPacket -> fee.SendPacket -> callbacks.SendPacket -> channel.SendPacket
+	// transferKeeper.sendTransfer -> callbacks.SendPacket -> feeKeeper.SendPacket -> channel.SendPacket
```

Comment:
might be best considering its the public name available

```suggestion
	// transferKeeper.Transfer -> callbacks.SendPacket -> feeKeeper.SendPacket -> channel.SendPacket
```

Reply:
I'd be happy with keeping it as `SendPacket` to avoid any confusion